lib/src/application_support_directory.dart

```diff
@@ -0,0 +1,8 @@
+// @dart=2.9
```

Comment:
I think you want to ignore this file as it gives an analyzer error:

``// ignore_for_file: import_of_legacy_library_into_null_safe
``

Reply:
Thank you! It works!